cmd/utils/flags.go

```diff
@@ -131,6 +131,10 @@ var (
 		Name:  "usb",
 		Usage: "Enable monitoring and management of USB hardware wallets",
 	}
+	RangeLimitFlag = cli.BoolFlag{
+		Name:  "rangelimit",
+		Usage: "Enable 5000 blocks limit for range query",
```

Comment:
Instead of introducing a bool flag, we should add an integer flag and the limit can be specified by users. The default value can be 0 which means no limitation.

Reply:
Modified

Reply:
Text needs update now, since it's not a 5000 bool anymore